src/modules/recent_scores.js

```diff
@@ -1,8 +1,13 @@
-const SCORES = [
-  { name: 'Name', score: 100 }, { name: 'Name', score: 20 },
-  { name: 'Name', score: 20 }, { name: 'Name', score: 78 },
-  { name: 'Name', score: 125 }, { name: 'Name', score: 77 },
-  { name: 'Name', score: 42 },
-];
+import {
+  baseUrl, games, id, scores,
+} from './api.js';
 
-export default SCORES;
\ No newline at end of file
+const path = `${baseUrl}${games}${id}${scores}`;
+
+async function recentScores() {
```

Comment:
- [ ] Please try to prefer using the newer ES6 syntax. In this case please switch this `function` syntax` with an arrow function
```suggestion
const recentScores = async () => {
```